components/google-cloud/dependencies.py

```diff
@@ -0,0 +1,34 @@
+# Lint as: python3
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
nit: 2021

Reply:
Same below. Unless these files appeared in other OSS'ed repos before :)

Reply:
done.